library/psa_crypto_pake.c

```diff
@@ -216,7 +223,20 @@ psa_status_t mbedtls_psa_pake_setup(mbedtls_psa_pake_operation_t *operation,
 
     operation->password = mbedtls_calloc(1, password_len);
     if (operation->password == NULL) {
-        return PSA_ERROR_INSUFFICIENT_MEMORY;
+        status = PSA_ERROR_INSUFFICIENT_MEMORY;
+        goto error;
```

Comment:
I believe that a `goto` is not needed at this point, but it should be harmless.

Reply:
We could have returned here, but I think that both versions are acceptable.